<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1121</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="cwLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="EmptyPage"/>
       <widget class="QWidget" name="PersonsListPage">
        <widget class="QTableView" name="PersonsTable">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>971</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatenbank">
    <property name="title">
     <string>DK Datenbank</string>
    </property>
    <addaction name="action_Neue_DB_anlegen"/>
    <addaction name="actionDBoeffnen"/>
    <addaction name="actionDB_schliessen"/>
    <addaction name="separator"/>
    <addaction name="actionProgramm_beenden"/>
   </widget>
   <widget class="QMenu" name="menuDK_Geber">
    <property name="title">
     <string>DK Geber</string>
    </property>
    <addaction name="action_Liste"/>
   </widget>
   <widget class="QMenu" name="menuVertr_ge">
    <property name="title">
     <string>Verträge</string>
    </property>
    <addaction name="actionListe_der_Vertr_ge_anzeigen"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <addaction name="actioncreateSampleData"/>
   </widget>
   <addaction name="menuDatenbank"/>
   <addaction name="menuDK_Geber"/>
   <addaction name="menuVertr_ge"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Neue_DB_anlegen">
   <property name="text">
    <string>&amp;Neue Anlegen</string>
   </property>
   <property name="toolTip">
    <string>Legen Sie hiermit eine leere DK Datenbank an</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDBoeffnen">
   <property name="text">
    <string>&amp;Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Öffnen Sie hiermit eine existierende Datenbank</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDB_schliessen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Schließen</string>
   </property>
   <property name="toolTip">
    <string>Schließen Sie hiermit die geöffnete Datenbank</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Liste">
   <property name="text">
    <string>&amp;Liste</string>
   </property>
   <property name="toolTip">
    <string>Zeigt eine Liste aller DK Geber an</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionListe_der_Vertr_ge_anzeigen">
   <property name="text">
    <string>Liste der Verträge anzeigen</string>
   </property>
  </action>
  <action name="actioncreateSampleData">
   <property name="text">
    <string>createSampleData</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
